### DISCUSSION TITLE
Issue: (Links to MDN resources like code on github don't open on new tab)

### DISCUSSION THREAD

**Discussion by FitseTLT** [mdn/yari#5566] [Ideas]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Structuring_a_page_of_content

#### What information was incorrect, unhelpful, or incomplete?
Most Links to MDN or other resources don't open in new tab by default and I need to press Ctrl every time I click a link. This is not common and logical behaviour the reader shouldn't be distracted whenever they click on links.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/html/introduction_to_html/structuring_a_page_of_content`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Structuring_a_page_of_content
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/html/introduction_to_html/structuring_a_page_of_content/index.md
* Last commit: https://github.com/mdn/content/commit/0160aac0e623dacdce4019205f32f7b3d3e58019
* Document last modified: 2022-02-18T09:33:58.000Z

</details>

**Comment by schalkneethling** [comment]
@FitseTLT, thank you for opening the discussion around this. We have talked about this before internally. We made an intentional choice not to open links in a new tab/window for two reasons:

1. We leave it up to the reader to decide what they prefer on a case-by-case basis instead of forcing a choice on them. As you mentioned, holding down ctrl or cmd will open links in a new tab
2. It is not a great/ideal experience for people using assistive technologies

I hope this make sense to you as well. Thanks again.

**Comment by FitseTLT** [comment]
May be the second reason makes sense. Otherwise, in most websites, especially this kind of tutorial websites, have links which open in new tab by default so that the page you were reading is kept intact to get back to it. Nevertheless, I was just reporting a discomfort I am experiencing as a user, what matters is your decision after all.

**Comment by schalkneethling** [comment]
Hey @FitseTLT,

I converted this to. a discussion. Let’s see what the feedback is from the community. If it looks like there is enough folks for whom this would be the preferred experience, we could reevaluate our decision will taking accessibility into account. Thank you for getting the conversation going.
